src/components/RecordsSummary/RecordsSummary.js

```diff
@@ -0,0 +1,165 @@
+
+const ResourceTypeRow = ({ resourceType, resources, resourceIdsGroupedByType }) => {
+  const consolidatedIds = Object.values(resourceIdsGroupedByType[resourceType])
+    .reduce((acc, cur) => {
+      acc.push(...cur);
+      return acc;
+    }, []);
+  const inflatedObjects = pick(consolidatedIds, resources);
+  const [, latestDate] = getDataRange(inflatedObjects, 'Y');
+  return (
+    <View style={styles.resourceTypeRow}>
+      <Text style={styles.resourceName}>{RESOURCE_TYPES[resourceType]}</Text>
+      <Text style={styles.resourceCount}>{consolidatedIds.length}</Text>
+      <Text style={styles.resourceLatestDate}>{latestDate}</Text>
+    </View>
+  );
+};
+
+ResourceTypeRow.propTypes = {
+  resourceType: string.isRequired,
+  resources: instanceOf(Object).isRequired,
+  resourceIdsGroupedByType: instanceOf(Object).isRequired,
+};
+
+const RecordsSummary = ({
+  resourceIdsGroupedByType, resources,
+}) => {
+  const recordsTotal = getRecordsTotal(resources);
+
+  return (
+    <View style={styles.recordSummaryContainer}>
+      <View style={styles.recordsHeader}>
+        <Text style={styles.recordsHeaderText}>
+          Records
+        </Text>
+        <Text style={styles.recordsHeaderTotal}>
+          {recordsTotal}
+          {' '}
+          Total
+        </Text>
+      </View>
+      <View style={styles.resourceTypeContainer}>
+        <View style={styles.resourceTypeRow}>
+          <Text style={styles.resourceName} />
+          <Text style={styles.resourceCountLabel}>count</Text>
+          <Text style={styles.resourceLatestDateLabel}>newest</Text>
+        </View>
+        {Object.entries(resourceIdsGroupedByType).map(
+          ([resourceType]) => (
+            <ResourceTypeRow
+              key={resourceType}
+              resourceType={resourceType}
+              resources={resources}
+              resourceIdsGroupedByType={resourceIdsGroupedByType}
+            />
+          ),
+        )}
+      </View>
+    </View>
+  );
+};
+
+RecordsSummary.propTypes = {
+  resourceIdsGroupedByType: shape({}),
+  resources: shape({}),
+};
+
+RecordsSummary.defaultProps = {
+  resourceIdsGroupedByType: {},
+  resources: null,
+};
+
+const mapStateToProps = (state) => ({
+  resources: state.resources,
+  resourceIdsGroupedByType: supportedResourcesSelector(state),
+});
+
+const mapDispatchToProps = { clearPatientDataAction: clearPatientData };
+
+export default connect(mapStateToProps, mapDispatchToProps)(RecordsSummary);
+
+const styles = StyleSheet.create({
+  recordSummaryContainer: {
+    marginTop: 10,
+    marginBottom: 20,
+    marginHorizontal: 20,
+    justifyContent: 'center',
+  },
+  recordsHeader: {
+    padding: 5,
+    backgroundColor: Colors.secondary,
+    flexDirection: 'row',
+  },
+  recordsHeaderText: {
+    color: 'white',
+    fontSize: 16,
+    padding: 5,
+  },
+  recordsHeaderTotal: {
+    color: 'white',
+    fontSize: 12,
+    padding: 9, // FIXME, not the right way to align
```

Comment:
Could use advice on this one - is it correct or should another way be used?

Reply:
I would probably use padding, but this is fine.